cmd/pctl/upgrade.go

```diff
@@ -18,8 +19,12 @@ func upgradeCmd() *cli.Command {
 	return &cli.Command{
 		Name:      "upgrade",
 		Usage:     "upgrade profile installation",
-		UsageText: "To upgrade an installation: pctl upgrade pctl-profile-installation-path/ v0.1.1 ",
-		Flags:     createPRFlags,
+		UsageText: "To upgrade an installation: pctl upgrade pctl-profile-installation-path/ v0.1.1",
+		Flags: append(createPRFlags, &cli.BoolFlag{
+			Name:        "latest",
+			Usage:       "--latest",
+			DefaultText: "*WARNING*: Upgrade to the latest version.",
```

Comment:
```suggestion
			DefaultText: "*WARNING*: Upgrade to the latest version. Please ensure this is a valid upgrade path before proceeding",
```

Reply:
I had `; whatever that is.` Yours is definitely more polite hahahahahahhaa.